src/main/java/net/atos/zac/app/zaken/model/RESTZaak.java

```diff
@@ -11,22 +11,27 @@
 import java.util.Map;
 import java.util.UUID;
 
+import jakarta.validation.constraints.NotNull;
+
 import net.atos.zac.app.identity.model.RESTGroup;
 import net.atos.zac.app.identity.model.RESTUser;
 import net.atos.zac.app.klanten.model.klant.IdentificatieType;
 import net.atos.zac.app.policy.model.RESTZaakRechten;
 import net.atos.zac.zoeken.model.ZaakIndicatie;
 
+
 public class RESTZaak {
 
     public UUID uuid;
 
     public String identificatie;
 
-    public String omschrijving;
+    @NotNull(message = "Omschrijving is required")
+    public String omschrijving; // Assuming this is the description field
 
     public String toelichting;
 
+    @NotNull(message = "Zaaktype is required")
```

Comment:
I would leave out these messages? I don't think they add anything and the danger is that they will grow out of sync with the parameters. Or did you find these messages are needed for a proper error message?

Reply:
Not quite true I think because from the 400 error message you can extract the field in question right?  E.g.

```    "path": "createZaak.arg0.zaak.groep.id",```

The problem with generating error msgs in the backend is that they are harder to maintain in the backend. Much easier to do this in the frontend. Also if you do this in the backend you get a possible future issue with translations.

Reply:
Too bad, already removed them